### ISSUE TITLE
Links generated with :rfc: do not use TLS

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1372]
They should be point at the "https" version of the URL. I believe this patch fixes the issue:

``` diff
diff -r fac83bacaed5 sphinx/environment.py
--- a/sphinx/environment.py Sun Feb 02 08:05:59 2014 +0100
+++ b/sphinx/environment.py Mon Feb 03 10:07:54 2014 -0800
@@ -59,7 +59,7 @@
     'embed_stylesheet': False,
     'cloak_email_addresses': True,
     'pep_base_url': 'http://www.python.org/dev/peps/',
-    'rfc_base_url': 'http://tools.ietf.org/html/',
+    'rfc_base_url': 'https://tools.ietf.org/html/',
     'input_encoding': 'utf-8-sig',
     'doctitle_xform': False,
     'sectsubtitle_xform': False,
diff -r fac83bacaed5 tests/test_build_html.py
--- a/tests/test_build_html.py  Sun Feb 02 08:05:59 2014 +0100
+++ b/tests/test_build_html.py  Mon Feb 03 10:07:54 2014 -0800
@@ -129,7 +129,7 @@
         (".//li/tt/em/span[[False](https://bitbucket.org/class)='pre']", '^i$'),
         (".//a[[False](https://bitbucket.org/class)='http://www.python.org/dev/peps/pep-0008']"
             "[[False](https://bitbucket.org/class)='pep reference external']/strong", 'PEP 8'),
-        (".//a[[False](https://bitbucket.org/class)='http://tools.ietf.org/html/rfc1.html']"
+        (".//a[[False](https://bitbucket.org/class)='https://tools.ietf.org/html/rfc1.html']"
             "[[False](https://bitbucket.org/class)='rfc reference external']/strong", 'RFC 1'),
         (".//a[[False](https://bitbucket.org/class)='objects.html#envvar-HOME']"
             "[[False](https://bitbucket.org/class)='reference internal']/tt/span[[False](https://bitbucket.org/class)='pre']", 'HOME'),
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1372
- Originally reported by: Alex Gaynor
- Originally created at: 2014-02-03T19:08:36.147

**Comment by shimizukawa** [comment]
_From Alex Gaynor on 2014-11-17 17:30:15+00:00_

Pining on this; I think this patch should still work. Let me know if there's anything else you need from me.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-11-17 17:43:33+00:00_

Hmm, guess we missed this patch.

I did the same change for the PEPs URL while at it.

**Comment by shimizukawa** [comment]
_From Alex Gaynor on 2014-11-17 18:03:08+00:00_

Is the change pushed?

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-11-17 18:36:56+00:00_

Closes #1372: use https URL for PEPs and RFCs

→ <<cset 571cae87a5a07cc2f2d920e6b6eb16d8cfbb56c6>>

**Comment by shimizukawa** [comment]
_From Alex Gaynor on 2014-11-17 18:42:36+00:00_

Thanks!
